frontend/src/v4/routes/components/resources/attachResourcesDialog/attachResourcesDialog.styles.tsx

```diff
@@ -46,17 +47,25 @@ export const StyledDropZone = styled(Dropzone)`
 	margin-bottom: 0;
 `;
 
-export const ResourcesListContainer = styled.div`
+export const ResourcesListContainerComponent = styled.div`
 	overflow: hidden;
 	width: 500px;
 	& >:last-child {
 		height: 55px;
 	}
 `;
 
+export const ResourcesListContainer = ({ children }) => (
+	<ViewerScrollArea autoHeight autoHeightMax={isV5() ? 200 : 114}>
+		<ResourcesListContainerComponent>
+			{children}
+		</ResourcesListContainerComponent>
+	</ViewerScrollArea>
+);
+
```

Comment:
Instead of adding a new component can we make it work by changing ResourcesListScroller to be a ViewerScrollArea 

This is a place where ResourcesListScroller  is being used:
<img src="https://user-images.githubusercontent.com/42032433/175514966-5b670f97-6245-43e0-8663-c51c8f2ebee3.png" width="400"/>

On a hunch I think it could work and make more sense semantically

Reply:
Done